pkg/apis/app/v2/kieapp_types.go

```diff
@@ -229,6 +229,8 @@ type KieAppObject struct {
 	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
 	// Keystore secret name
 	KeystoreSecret string `json:"keystoreSecret,omitempty"`
+	// The image context to use
+	ImageContext string `json:"imageContext,omitempty"`
 	// The image to use
 	Image string `json:"image,omitempty"`
 	// The image tag to use
```

Comment:
Could you please add more specific description for these parmas? Like that this parameter is optional, in what cases is needed to be set, example value.
I think we can copy description from UI here, then it would be visible also in crd, as crd is generated from api :)

Reply:
Done